### PR TITLE
Added enum example in variables section.

### PR DESCRIPTION
**Pull request by lucavictor220**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -243,6 +243,38 @@ function loadPages(count: number = 10) {
...
+}
+
+const comedy = async requestFilm(GENRE.COMEDY);
+```
+
+**Good:**
+
+```ts
+enum GENRE {
+  ROMANTIC = 'Romantic',
```
Comment (dimadeveatii) [inline README.md]: What is the motivation behind this enum being string-based?  
It is then arguable that it has advantage over bad example with just constants.

Would it be better maybe to use [constant member](https://www.typescriptlang.org/docs/handbook/enums.html#computed-and-constant-members), like:
```ts
enum Genre {
  Romantic,
  Drama,
  ...
}
```

**Comment by dimadeveatii** [comment]
Please fix above notes.

**Review by dimadeveatii — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lucavictor220) [reply README.md]: I rethought the intent and motivation of enum.

File: README.md [hunk 2]
```diff
@@ -243,6 +243,59 @@ function loadPages(count: number = 10) {
...
+}
+
+projector.configureFilm(GENRE.COMEDY);
+
+class Projector {
+  // delactation of Projector
+  configureFilm(genre) {
+    switch (genre) {
+      case GENRE.ROMANTIC:
+        // some logit to be executed
```
Comment (dimadeveatii) [inline README.md]: Typo: `logit`

File: README.md [hunk 3]
```diff
@@ -243,6 +243,59 @@ function loadPages(count: number = 10) {
...
```
Comment (dimadeveatii) [inline README.md]: Same typo `logit`

**Review by dimadeveatii — CHANGES_REQUESTED** [review]
Much better now!
Please fix only the small typo I suggested in comments.

File: README.md (hunk 2 above)
Comment (lucavictor220) [reply README.md]: Done!

**Review by dimadeveatii — APPROVED** [review]

**Pull request merged**